.github/workflows/measure-packages-size.yml

```diff
@@ -1,45 +0,0 @@
-name: Measure packages size
```

Comment:
I liked this buddy from time to time, and we are not using this workflow often. Not relevant for this PR, but what do you think to introduce package changes to releases?

Reply:
I like it too, but it uses an action from a fork owned by someone who left the team/company, so we can't use it as-is for now.

Additionally, it uses SAM to bundle which in turns uses Docker. Now that we have a better understanding of `esbuild` (see Layers work) and headless CDK (see `testing` package) we can do the same with local bundling and measure that, which is much simpler and faster.

We'll reintroduce this in the near-ish future, how soon depends on how the quarter/year planning goes in the coming days & weeks.

Reply:
agree